util/bump_version.sh

```diff
@@ -48,13 +48,11 @@ back_to_devel() {
 
 	sed -i -re "s/^set\(VERSION_PATCH [0-9]+\)$/set(VERSION_PATCH $NEXT_VERSION_PATCH)/" CMakeLists.txt
 
-	sed -i -re "1s/[0-9]+\.[0-9]+\.[0-9]+/$NEXT_VERSION/g" doc/lua_api.txt
-
 	sed -i -re "1s/[0-9]+\.[0-9]+\.[0-9]+/$NEXT_VERSION/g" doc/menu_lua_api.txt
 
 	sed -i -re "1s/[0-9]+\.[0-9]+\.[0-9]+/$NEXT_VERSION/g" doc/client_lua_api.md
```

Comment:
Should now be 'client_lua_api.txt', i didn't realise i had to change this code too.

Reply:
This isn't related to this PR.

Reply:
Sorry, unrelated indeed and should be a separate commit, you don't have to do it.